apps/nowcasting-app/components/layout/header/profile-dropdown.tsx

```diff
@@ -59,7 +60,10 @@ const ProfileDropDown = ({}: IProfileDropDown) => {
                 className={"text-right"}
                 fullWidth
               >
-                <a href="mailto:ops@openclimatefix.org">Help</a>
+                <a href="mailto:ops@openclimatefix.org">
+                  Help{" "}
+                  <ExternalLinkIcon className="inline-block w-3 h-3 ml-[6px] mb-[1px] self-center text-inherit color-black" />
```

Comment:
Is fine here, but think these general notes/changes should be PR'd into `staging` and then merged back into branches 👍